src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1394 @@
+            doc="""Indicates whether this model will be dynamic or not,
+    **default** = False. """,
+        ),
+    )
+    CONFIG.declare(
+        "has_holdup",
+        ConfigValue(
+            default=False,
+            domain=In([False]),
+            description="Holdup construction flag - must be False",
+            doc="""Indicates whether holdup terms should be constructed or not.
+    **default** - False.""",
+        ),
+    )
+    CONFIG.declare(
+        "property_package_water",
+        ConfigValue(
+            default=useDefault,
+            domain=is_physical_parameter_block,
+            description="Property package to use for feed, distillate, brine and cooling water properties",
+            doc="""Property parameter object used to define water property calculations,
+    **default** - useDefault.
+    **Valid values:** {
+    **useDefault** - use default package from parent model or flowsheet,
+    **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    CONFIG.declare(
+        "property_package_steam",
+        ConfigValue(
+            default=useDefault,
+            domain=is_physical_parameter_block,
+            description="Property package to use for heating and motive steam properties",
+            doc="""Property parameter object used to define steasm property calculations,
+    **default** - useDefault.
+    **Valid values:** {
+    **useDefault** - use default package from parent model or flowsheet,
+    **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    CONFIG.declare(
+        "property_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property packages",
+            doc="""A ConfigBlock with arguments to be passed to a property block(s)
+    and used when constructing these,
+    **default** - None.
+    **Valid values:** {
+    see property package for documentation.}""",
+        ),
+    )
+
+    def build(self):
+        super().build()
+
+        self.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+        units_meta = self.config.property_package_water.get_metadata().get_derived_units
+
+        """
+        Add system configurations
+        """
+        self.number_effects = Param(
+            initialize=12,
+            mutable=True,
+            within=Set(initialize=[8, 10, 12, 14, 16]),
+            units=pyunits.dimensionless,
+            doc="Number of effects",
+        )
+
+        self.delta_T_last_effect = Param(
+            initialize=10,
+            mutable=True,
+            units=pyunits.K,
+            doc="Temperature increase in last effect",
+        )
+
+        self.delta_T_cooling_reject = Param(
+            initialize=-3,
+            mutable=True,
+            units=pyunits.K,
+            doc="Temperature decrease in cooling reject water",
+        )
```

Comment:
Why not make these variables? We could optionally fix them within this build method and then the user would have the option to unfix as desired and solve for them.

Reply:
Unless these are set in the surrogate model and aren't meant to be changed. However, if that were the case, maybe these parameters shouldn't be mutable.

Reply:
I wonder if we want to keep this flexible and make them variables that subsequently get fixed. At a minimum, it would be useful to add comments above each Param clarifying that these were parameters attributed to a particular configuration of which the surrogate model is based on.

Reply:
Actually I think you both make sense. Physically it should be a variable, but then we want to fix them right after the declaration, and add comments about it. I think I want to do it in a separate PR.